doc/api/errors.md

```diff
@@ -2352,6 +2352,11 @@ The `--entry-type=...` flag is not compatible with the Node.js REPL.
 Used when an [ES Module][] loader hook specifies `format: 'dynamic'` but does
 not provide a `dynamicInstantiate` hook.
 
+<a id="ERR_OPTION_INCOMPATIBLE_WITH_PLATFORM"></a>
+#### ERR_OPTION_INCOMPATIBLE_WITH_PLATFORM
+
+Used when a function is called,
+with an incompatible option to the current platform which is running Node.js.
```

Comment:
I truly believe that an error that is related to unsupported feature on a platform should be. Whether it should be specific to options or not is debatable.
I think it should, because an option parameter is a pattern the reoccurs throughout the API, and as far as I can tell, it is almost certain that function which is platform dependent will never be part of the API.
So I understand what you say, but I think we should re-consider this.

Reply:
the comma will be fixed. thanks.

Reply:
hmm, I re thought about this. fs.chown and fs.chmod are also platform dependent. And as such should also throw exceptions (but handling this is out of the scope of this PR).
Because of this reason I now support ERR_FEATURE_UNAVAILABLE_ON_PLATFORM .
Should we wait for more opinions?

Reply:
fixed NITS, also error is updated.